joomla/joomla-cms
# [3.7.x] Convert installation template to html5 (also move inline scripts to js)

### Summary of Changes

Convert installation template to html5, also move inline scripts to js
### Testing Instructions
1. Code review
2. Apply patch in 3.7.x branch
3. Delete configuration.php
4. Install joomla as usual and confirm install works
### Documentation Changes Required

None.
